src/components/CodeInput/PCodeInput.vue

```diff
@@ -132,18 +132,19 @@
 }
 
 .p-code-input,
-.p-code-input__view,
-.p-code-input__control,
-.p-code-input__textarea,
-.p-code-input__line-numbers-wrapper {
+.p-code-input .p-code-input__view,
+.p-code-input .p-code-input__control,
+.p-code-input .p-code-input__textarea,
+.p-code-input .p-code-input__line-numbers-wrapper {
```

Comment:
Why the increase in specificity here? We shouldn't need to do this for BEM classes

Reply:
So here we're adding styles for other components, the flat specificity appeared to not be overriding the style declarations in the components. Why was this build this way? There's a lot of small questions like this that I'll continue to encounter, haha. I will take another look at this though.

Reply:
Hm I'm not sure what you mean by other components, most of these are declared in the template but let's pair on this so I can explain how it works if you get stuck, this is one of the more complicated components cause it's doing some interesting stuff (like the invisible input + overlay)

Reply:
Hey my comment there was clearly burnt-out-Friday-evening. I’ll take a closer look at this for sure, will reach out if I’m confused. It probably just needs to be moved down the file rather than increased selectors.